src/Tizen.NUI.Components/Controls/Toast.cs

```diff
@@ -76,19 +75,9 @@ public static Toast FromText(string text, uint duration)
 
         private Window window = null;
 		protected TextLabel[] textLabels = null;
-
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected TextLabel textLabel = null;
-
         private string strText = null;
         private NPatchVisual toastBackground = null;
```

Comment:
View has Background. (Property.Map)
I think it is not necessary to have npatch visual in Toast.